pkg/cmd/server/origin/master.go

```diff
@@ -1063,6 +1065,35 @@ func (c *MasterConfig) RunSecurityAllocationController() {
 	controller.Run()
 }
 
+func (c *MasterConfig) BootstrapSecurityContextConstraints() {
+	sccList, err := c.KubeClient().SecurityContextConstraints().List(labels.Everything(), fields.Everything())
+	if err != nil {
+		glog.Fatalf("Unable to initialize security context constrints: %v", err)
+	}
+	if len(sccList.Items) > 0 {
+		return
+	}
+	glog.Infof("No security context constraints detected, adding defaults")
+	scc := &kapi.SecurityContextConstraints{
+		ObjectMeta: kapi.ObjectMeta{
+			Name: "scc-default",
+		},
+		AllowPrivilegedContainer: true,
```

Comment:
Nothing selects a default constraint by name, a constraint becomes the default by virtue of having all users/groups mapped to it